db/migrate/20220321175733_create_player_stats.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePlayerStats < ActiveRecord::Migration[7.0]
+  def change
+    create_table :player_stats do |t|
+      t.integer :goals
+      t.integer :assists
+      t.integer :own_goals
```

Comment:
Te parece bien si los hacemos defaultear a 0?
```suggestion
      t.integer :goals, default: 0, null: false
      t.integer :assists, default: 0, null: false
      t.integer :own_goals, default: 0, null: false
```

Reply:
Yes!